admin-dev/themes/new-theme/.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
   rules: {
     'class-methods-use-this': 0,
     'func-names': 0,
+    'no-undef': 0,
```

Comment:
Ping @NeOMakinG

Reply:
Commented on the issue

Reply:
I think this solution won't be taken anyway, we want this rule